apps/sms/services/js/conversation/conversation_service.js

```diff
@@ -22,26 +38,74 @@ if (!('BridgeServiceMixin' in self)) {
   ]);
 
   var ConversationService = {
+    mobileMessageClient: null,
+
     /**
      * Initializes our service using the Service mixin's initService.
      */
     init() {
       this.initService();
+
+      this.mobileMessageClient = bridge.client(
+        'mozMobileMessageShim',
+        new BroadcastChannel('mobile-message-shim-channel')
```

Comment:
For this patch I'm going to leave this fixed name for the broadcast channel and think about multiple-instance cases in the follow, does it work for you?